.env.development

```diff
@@ -1,7 +1,2 @@
-# Default admin/district authorization for development
 REACT_APP_DHIS2_BASE_URL="http://localhost:8080/dhis"
-REACT_APP_DHIS2_AUTHORIZATION="Basic c3lzdGVtOlN5c3RlbTEyMw=="
 REACT_APP_TRACKER_CAPTURE_APP_PATH="http://localhost:8080/dhis/dhis-web-tracker-capture"
```

Comment:
This should probably not be provided as an environment variable - it can be dynamically generated using `config.baseUrl`, since the relative path is always the same.